controllers/enable_mesh.go

```diff
@@ -112,10 +112,15 @@ func serviceMeshIsNotEnabled(meta metav1.ObjectMeta) bool {
 }
 
 func (r *OpenshiftServiceMeshReconciler) findIstioIngress(ctx context.Context) (routev1.RouteList, error) {
+	meshNamespace := "istio-system"
+	if env, defined := os.LookupEnv("MESH_NAMESPACE"); defined {
+		meshNamespace = env
+	}
+
 	routes := routev1.RouteList{}
 	if err := r.List(ctx, &routes, &client.ListOptions{
 		LabelSelector: labels.SelectorFromSet(labels.Set{"app": "odh-dashboard"}),
-		Namespace:     MeshNamespace,
+		Namespace:     meshNamespace,
```

Comment:
Can we remove `MeshNamespace` const now?

Reply:
https://github.com/maistra/odh-project-controller/pull/24/commits/f84cdbd69e353511016d4c705c3ff509a7fd761f